globus_cli/services/transfer/share/create.py

```diff
@@ -9,15 +10,14 @@
 @click.command('create', help='Create a new Share, hosted on an Endpoint')
 @common_options
 @endpoint_create_and_update_opts(create=True, shared_ep=True)
-@click.option('--host-endpoint-id', required=True,
-              help='ID of the endpoint on which this Share is hosted')
-def share_create(host_endpoint_id, display_name, description, organization,
+@endpoint_id_arg(metavar='HOST_ENDPOINT_ID')
+def share_create(endpoint_id, display_name, description, organization,
                  contact_email, contact_info, info_link, public):
```

Comment:
We seem to be creating shares without letting the user specify the path on the host (which should be required).